smarts/core/sensors.py

```diff
@@ -1022,25 +1026,39 @@ def teardown(self):
 
 class AccelerometerSensor(Sensor):
     def __init__(self, vehicle, sim):
-        self.linear_accelerations = deque(maxlen=3)
-        self.angular_accelerations = deque(maxlen=3)
+        self._dt = sim.timestep_sec
+        self.linear_velocities = deque(maxlen=3)
+        self.angular_velocities = deque(maxlen=3)
 
-    def __call__(self, linear_velocity, angular_velocity):
+    def __call__(self, linear_velocity, angular_velocity, heading):
```

Comment:
Looks like heading is not used.

Reply:
Yep, good catch.  I had added it when I tried to do something a different way in the accelerometer, but then (incompletely) rolled that back.  removing.